advanced/Scripts/query.sh

```diff
@@ -58,12 +61,131 @@ scanList(){
     esac
 }
 
+# function to count new domain entries since last gravity run
+CountNewDomains() {
+  printf %s "  ${INFO} Running query, please be patient..."
+  # store query result (count) in variable
+  newdomainentries=$(
+    sqlite3 "${gravityDBfile}" << EOSQL
+      ATTACH '${gravityOLDfile}' AS old;
+      SELECT count(DISTINCT domain) FROM gravity WHERE domain NOT IN (SELECT domain FROM old.gravity);
+EOSQL
+      )
+  printf '\r%s\n' "$(tput el)  ${INFO} ${newdomainentries} new domain(s) found."
+  exit 0
+}
+
+# function to list new domain entries since last gravity run
+ListNewDomains() {
+  if [ -z "${all}" ]; then
+    QueryLimit="Limit 100"
+  fi
+  printf %s "  ${INFO} Running query, please be patient..."
+  # store query result (list) in array
+  # order by adlist.address makes the query slower, but the result is more useful
+  mapfile -t newdomainarray < <( sqlite3 "${gravityDBfile}" << EOSQL
+      ATTACH '${gravityOLDfile}' AS old;
+      SELECT domain, '' || adlist.address || '' FROM gravity
+      JOIN adlist ON adlist.id = gravity.adlist_id	  
+      WHERE domain NOT IN (SELECT domain FROM old.gravity)
+      ORDER BY adlist.address ${QueryLimit};
+EOSQL
+      )
+  if [ "${#newdomainarray[*]}" -eq 0 ]; then
+    printf '\r%s\n' "$(tput el)  ${INFO} No new domains have been found."
+    exit 0
+  else
+    printf '\r%s\n' "$(tput el)  ${INFO} ${#newdomainarray[*]} new domain(s) found."
+  fi
+  for (( j=0; j<"${#newdomainarray[@]}"; j++ )); do
+    IFS='|'
+    read -r domain urllist <<< "${newdomainarray[j]}"
+    echo -e "  ${domain} (${COL_GREEN}${urllist}${COL_NC})"
+  done
+  if [[ -z "${all}" ]] && [[ "${#newdomainarray[*]}" -ge 100 ]]; then
+    echo -e "  ${INFO} Over 100 new domains found.
+        This can be overridden using the -all option"
```

Comment:
Clarify what can be overwritten: the limit.

```suggestion
    echo -e "  ${INFO} Over 100 new domains found.
        If you want to see them all, append \"-all\" to the command"
```